packages/bpk-component-accordion/src/BpkAccordionItem.js

```diff
@@ -45,14 +45,13 @@ const BpkAccordionItem = props => {
     );
   }
 
-  const titleId = `${id}_title`;
   const contentId = `${id}_content`;
 
   return (
     <div id={id} {...rest}>
       <dt
-        role="heading"
         aria-level="3"
+        aria-labelledby={id}
```

Comment:
ARIA attributes must conform to valid values
`aria-labelledby` must match a valid element id
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute